Makefile

```diff
@@ -175,6 +176,8 @@ security-check: vendor
 
 deps: vendor web/built                                                                                 ## Install the project PHP and JS dependencies
 
+update: composer.lock                                                                                  ## Update composer.lock for php dependencies
```

Comment:
Un-needed, please remove it. The Makefile contains enough stuff.

If one wants to update dependencies he will just type the command by hand.